app/controllers/users_controller.rb

```diff
@@ -1,17 +1,31 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  skip_before_action :check_token, only: %i[create create_session]
+  skip_before_action :check_token, only: %i[create confirm_email create_session]
 
   def create
     @user = User.create(create_params)
     if @user.save
       render status: :created, action: :show
+      UserMailer.email_confirmation(@user, @user.email).deliver_now
```

Comment:
We should seriously consider a background job to avoid `deliver_now`

Reply:
Great idea! looking into it